src/task_run_diagnostic_reports.py

```diff
@@ -53,6 +58,21 @@ def success_callback(report, version):
         return {}
 
 
+def create_ticket(group, disease, touchstone,
+                  report: ReportConfig, version,
+                  yt: YTClient,
+                  config: Config):
+    try:
+        issue = yt.create_issue(Project("78-0"),
```

Comment:
This is the "VIMC" project code - seems fine to hard-code as can't envision this changing